transforms/language/html2parquet/kfp_ray/Makefile

```diff
@@ -2,10 +2,20 @@ REPOROOT=${CURDIR}/../../../../
 WORKFLOW_VENV_ACTIVATE=${REPOROOT}/transforms/venv/bin/activate
 include $(REPOROOT)/transforms/.make.workflows
 
+
 # Include the common configuration for this transform
-include ../transform.config
+#include ../transform.config
+
+SRC_DIR=${CURDIR}/../
+# Use the docker image that is built for ray runtime
+TRANSFORM_RUNTIME=ray
+## override settings in .make.default as they assume old structure with ray being the current folder
+DOCKER_IMAGE_NAME=$(TRANSFORM_NAME)-$(TRANSFORM_RUNTIME)
+DOCKER_LOCAL_IMAGE=$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_VERSION)
```

Comment:
DOCKER_LOCAL_IMAGE is defined the same in .make.defaults, so I think you can remove it here.

Reply:
no. this is a bit more complicated than this. Let's keep it for now. I will do a second pass on the makefile and trim some of the fat...